apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -608,7 +608,7 @@ const BookingPage = ({
                     {t("former_time")}
                   </p>
                   <p className="text-gray-500 line-through dark:text-white">
-                    <Icon.FiCalendar className="mr-[10px] ml-[2px] -mt-1 inline-block h-4 w-4 text-gray-400" />
+                    {/* <Icon.FiCalendar className="mr-[10px] ml-[2px] -mt-1 inline-block h-4 w-4 text-gray-400" /> */}
```

Comment:
I was just wondering why you left out this icon here, maybe you forgot to uncomment?

Reply:
yeah fixed now. was just testing something